cranelift/codegen/src/isa/x64/inst/emit_tests.rs

```diff
@@ -2408,21 +2408,6 @@ fn test_x64_emit() {
         "movsd   %xmm14, %xmm3",
     ));
 
-    // ========================================================
-    // XMM_R_R
-
-    insns.push((
-        Inst::xmm_r_r(SseOpcode::Movss, xmm3, w_xmm2),
-        "F30F10D3",
-        "movss   %xmm3, %xmm2",
-    ));
-
-    insns.push((
-        Inst::xmm_r_r(SseOpcode::Movsd, xmm4, w_xmm3),
```

Comment:
ditto here

Reply:
See above. This test is already included I just modified the registers as suggested by @julian-seward1